From e86b73738e26e35a17b690178e258e050272f422 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Wed, 25 Apr 2018 13:23:30 +0800
Subject: [PATCH 4/5] ls1021a: dts: change the eTSEC queque to 8

eTSEC can support 8 H/W queues, show this in the device tree.
Just like SDK Freescale-Linux-SDK-for-LS1021A-IOT-Rev2-v0.4-20150907-yocto.iso
has done.

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm/boot/dts/ls1021a-iot.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/ls1021a-iot.dts b/arch/arm/boot/dts/ls1021a-iot.dts
index 7556282..c657c56 100644
--- a/arch/arm/boot/dts/ls1021a-iot.dts
+++ b/arch/arm/boot/dts/ls1021a-iot.dts
@@ -110,6 +110,8 @@
 	phy-handle = <&phy1>;
 	phy-connection-type = "sgmii";
 	fsl,dma-endian-le;
+	fsl,num_rx_queues = <0x8>;
+	fsl,num_tx_queues = <0x8>;
 	status = "ok";
 };
 
@@ -118,6 +120,8 @@
 	phy-handle = <&phy3>;
 	phy-connection-type = "sgmii";
 	fsl,dma-endian-le;
+	fsl,num_rx_queues = <0x8>;
+	fsl,num_tx_queues = <0x8>;
 	status = "ok";
 };
 
@@ -126,6 +130,8 @@
 	fixed-link = <2 1 1000 0 0>;
 	phy-connection-type = "rgmii";
 	fsl,dma-endian-le;
+	fsl,num_rx_queues = <0x8>;
+	fsl,num_tx_queues = <0x8>;
 	status = "ok";
 };
 
-- 
1.8.5.2.233.g932f7e4

